src/pages/Background/index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title></title>
-  </head>
 
-  <body></body>
-</html>
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
```

Comment:
Do not make any changes to html/css files
Also adding links can cause Content Security Policy issues

Try looking for how to add fonts in react apps or keep the default font. Upto you